packages/gatsby-source-contentful/src/__tests__/gatsby-node.js

```diff
@@ -372,6 +374,8 @@ describe(`gatsby-node`, () => {
         "contentful-sync-data-testSpaceId-master",
       ]
     `)
+
+    expect(reporter.info.mock.calls).toMatchSnapshot()
```

Comment:
wouldn't it be better to just count the createNode calls instead of the reporter data?

Reply:
I'd vote to do both, checking node creation count first.

That way we catch changes in the log output.

Reply:
👍 but inline snapshot might be better :)

Reply:
ohh yes!